cleanlab/count.py

```diff
@@ -753,10 +762,27 @@ def estimate_confident_joint_and_cv_pred_proba(
                 holdout_inds = np.where(s_holdout_cv == missing_class)[0]
                 # Duplicate one instance of missing_class from holdout data to the training data:
                 dup_ind = holdout_inds[0]
-                s_train_cv = np.append(s_train_cv, s_holdout_cv[dup_ind])
-                X_train_cv = np.vstack([X_train_cv, X_holdout_cv[dup_ind]])
+                s_train_cv = np.append(
+                    s_train_cv, s_holdout_cv[dup_ind]
+                )  # labels are always np.array
+                if isinstance(X, np.ndarray):
+                    X_train_cv = np.vstack([X_train_cv, X_holdout_cv[dup_ind]])
+                else:
+                    try:
+                        if isinstance(X, pd.DataFrame):
+                            X_extra = X_holdout_cv.iloc[dup_ind]
+                        elif isinstance(X, pd.Series):
+                            X_extra = pd.Series(X_holdout_cv.iloc[dup_ind])
+                        else:
+                            X_extra = X_holdout_cv[dup_ind]
+                        X_train_cv = X_train_cv.append(X_extra)
+                    except:
+                        raise TypeError("Features object X must support: X.append(X[i])")
```

Comment:
I think typically the training "dataset" object would typically contain labels, so prefer "features"?
Especially to prepare this library for eventually also identifying issues in the features, it would be good to standardize the terminology.   I think "dataset issues" terminology will be reserved for the existing `cleanlab.dataset` module.

Just FYI: 
- Current docstring in CleanLearning.fit() called X the "Input feature matrix".
- Sklearn calls `X` the "training input", which I don't prefer as a terminology.

Reply:
I like:
x = data or feature matrix or data features 
y = labels

Reply:
changed to "Data features"